signaling-server/tests/test_room_manager.py

```diff
@@ -1,10 +1,21 @@
 import asyncio
 import unittest
 
+from typing import Optional
+
 from room_manager import RoomManager
 
-def _run_async(coroutine):
-    return asyncio.get_event_loop().run_until_complete(coroutine)
+
+class AsyncRunner:
+    def __init__(self, event_loop: Optional[asyncio.BaseEventLoop] = None) -> None:
+        self.loop = event_loop or asyncio.new_event_loop()
+
+    def run_async(self, coroutine):
+        return self.loop.run_until_complete(coroutine)
+
+    def __del__(self):
+        self.loop.close()
+
```

Comment:
get_event_loop deprecated in python3.10 for use outside of an event loop

Reply:
Il existe cela : https://docs.python.org/3/library/asyncio-task.html#asyncio.run

Reply:
`This function always creates a new event loop and closes it at the end. It should be used as a main entry point for asyncio programs, and should ideally only be called once.`
This reuses the same event loop, which might be a little less overhead

Reply:
Now that I understand, it's fine for me like that!